frontend/public/components/instantiate-template.tsx

```diff
@@ -251,7 +251,22 @@ class TemplateForm_ extends React.Component<
     }
 
     const template: TemplateKind = obj.data;
-    const parameters = template.parameters || [];
+
+    let parameters = template.parameters || [];
+    if (template?.metadata?.labels['workload.template.kubevirt.io/saphana'] === 'true') {
```

Comment:
you should not need to change the parameters.

if the parameters are wrong, ask to fix it in common template repo, we should show the template parameters as is.
if it's not currect we should fix it upsrream https://github.com/kubevirt/common-templates/pull/384

Reply:
@rmohr @dominikholler hi, sorry to be late to the party, after the template is merged.

It looks like we need to add `displayName` in the template parameters info [1] , o/w the UI will print the parameter name as title, can you add it to the template ?

[1] https://docs.openshift.com/container-platform/3.7/rest_api/apis-template.openshift.io/v1.TemplateInstance.html#object-schema